trview.app/Windows/TriggersWindow.cpp

```diff
@@ -314,6 +314,21 @@ namespace trview
                     add_stat("Flags", format_binary(selected_trigger->flags()));
                     add_stat("Only once", selected_trigger->only_once());
                     add_stat("Timer", selected_trigger->timer());
+
+                    ImGui::TableNextColumn();
+                    ImGui::Text("Colour");
+                    ImGui::TableNextColumn();
+
+                    if (ImGui::Button("Reset"))
+                    {
+                        selected_trigger->set_colour(std::nullopt);
+                    }
+                    ImGui::SameLine();
+                    DirectX::SimpleMath::Color colour = selected_trigger->colour();
+                    if (ImGui::ColorEdit4("##colour", &colour.x, ImGuiColorEditFlags_NoInputs))
+                    {
+                        selected_trigger->set_colour(colour);
+                    }
```

Comment:
Does this need unit tests? You only made them for lua.

Reply:
Yes I'll add some. The ColorEdit4 one might be more difficult though.

Reply:
Figuring out how to automate the color picker is too hard for now, so I left that one out.